ita-dash.json

```diff
@@ -1,1834 +1,1752 @@
-          "graphMode": "none",
-          "justifyMode": "auto",
-          "mode": "",
-          "orientation": "horizontal",
-          "reduceOptions": {
-            "calcs": [
-              "lastNotNull"
-            ],
-            "fields": "",
-            "values": false
-          },
-          "textMode": "auto"
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "8.4.2",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "exemplar": true,
+          "expr": "java_lang_OperatingSystem_AvailableProcessors{product=\"jira\"}",
+          "interval": "",
+          "legendFormat": "{{instance}}",
+          "refId": "A"
+        }
+      ],
+      "title": "CPU Cores",
+      "type": "stat"
+    },
+    {
+      "gridPos": {
+        "h": 11,
+        "w": 3,
+        "x": 12,
+        "y": 0
+      },
+      "id": 28,
+      "options": {
+        "alertInstanceLabelFilter": "",
+        "alertName": "",
+        "dashboardAlerts": true,
+        "dashboardTitle": "",
+        "groupBy": [],
+        "groupMode": "default",
+        "maxItems": 10,
+        "showOptions": "changes",
+        "sortOrder": 5,
+        "stateFilter": {
+          "alerting": false,
+          "error": true,
+          "execution_error": false,
+          "firing": true,
+          "inactive": false,
+          "noData": false,
+          "no_data": false,
+          "normal": false,
+          "ok": false,
+          "paused": false,
+          "pending": false
         },
-        "pluginVersion": "8.3.6",
-        "span": 0,
-        "sparkline": {},
-        "targets": [
-          {
-            "datasource": {
-              "type": "prometheus",
-              "uid": "u5eohEYnz"
-            },
-            "exemplar": true,
-            "expr": "java_lang_Runtime_StartTime{}",
-            "interval": "",
-            "legendFormat": "{{instance}}",
-            "refId": "A"
-          }
-        ],
-        "thresholds": "",
-        "title": "Start time",
-        "type": "stat",
-        "valueFontSize": "",
-        "valueName": ""
+        "tags": []
       },
-      {
-        "colorBackground": false,
-        "colorValue": false,
-        "decimals": 0,
-        "editable": false,
-        "error": false,
-        "fieldConfig": {
-          "defaults": {
-            "mappings": [],
-            "thresholds": {
-              "mode": "absolute",
-              "steps": [
-                {
-                  "color": "green",
-                  "value": null
-                },
-                {
-                  "color": "red",
-                  "value": 80
-                }
-              ]
+      "pluginVersion": "8.2.5",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "java_lang_OperatingSystem_AvailableProcessors{instance=\"localhost:2991\"}",
```

Comment:
There is still an instance reference here. Is that intentional?

Reply:
🦅 Well spotted! I've removed this.